CMakeLists.txt

```diff
@@ -68,7 +68,7 @@ set(MLC_LLM_RUNTIME_LINKER_LIB "")
 set(TOKENZIER_CPP_PATH 3rdparty/tokenizers-cpp)
 add_subdirectory(${TOKENZIER_CPP_PATH} tokenizers EXCLUDE_FROM_ALL)
 
-set(BUILD_PYTHON_BINDINGS ON)
+set(BUILD_PYTHON_BINDINGS OFF)
```

Comment:
we should change the flag ro XGRAMMAR_BUILD_PYTHON_BINDING to be generic

Reply:
I agree, and I have changed that